src/main/java/com/std/tothebook/api/controller/api/MyBookController.java

```diff
@@ -0,0 +1,46 @@
+package com.std.tothebook.api.controller.api;
+
+import com.std.tothebook.api.domain.dto.FindMyBookResponse;
+import com.std.tothebook.api.service.MyBookService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Tag(name = "독서기록")
+@RequestMapping("/api/myBook")
+@Controller
+@RequiredArgsConstructor
+public class MyBookController {
+
+    private final MyBookService myBookService;
+
+    @Operation(summary = "독서기록 리스트 조회")
+    @GetMapping("/")
```

Comment:
```suggestion
    @GetMapping("")
```
해당 url에 뒤에 더 들어갈 계층이 없다면 슬래시는 제외해주셔도 됩니다.